main.py

```diff
@@ -0,0 +1 @@
+print('Hello, world!')
```

Comment:
Hi! Welcome to the course.
All work concerning Lab1 must be located in the `lab_1_classify_by_unigrams` folder.
Please delete extra files.